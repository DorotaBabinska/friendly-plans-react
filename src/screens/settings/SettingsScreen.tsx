import React from 'react';

import { FullScreenTemplate } from 'components';
import { i18n } from 'locale';
import { headerStyle } from 'styles';
import { SignOutListItem } from './SignOutListItem';

export class SettingsScreen extends React.PureComponent {
  static navigationOptions = {
    title: i18n.t('settings:settings'),
    headerTitleStyle: headerStyle.headerText,
  };

  render() {
    return (
      <FullScreenTemplate>
        <SignOutListItem />
      </FullScreenTemplate>
    );
  }
}
