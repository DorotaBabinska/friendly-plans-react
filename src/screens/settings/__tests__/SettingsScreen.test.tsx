import { shallow } from 'enzyme';
import React from 'react';

import { SettingsScreen } from '../SettingsScreen';

describe('SettingsScreen', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<SettingsScreen />);
    expect(wrapper).toMatchSnapshot();
  });
});
