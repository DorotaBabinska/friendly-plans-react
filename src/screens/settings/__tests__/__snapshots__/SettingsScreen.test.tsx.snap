// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettingsScreen should match snapshot 1`] = `
<FullScreenTemplate>
  <SignOutListItem />
</FullScreenTemplate>
`;
